### ISSUE TITLE
[Question] is it ready to use "out of the box" ?

### ISSUE THREAD

**Issue by BlobIsBack** [IAmStoxe/wirehole#124]
I saw some pihole and wire guard tutorials and they said that we need to setup a firewall to make so that the server is "secure" so with this do I have to setup firewall and other stuff ?

**Comment by litex2x** [comment]
Nope. If you are using raspberry pi with omv installed, I can confirm this will not work out of box. It does give you a template to begin with. I see issues beyond my use case as well.
